README.md

```diff
@@ -1 +1,79 @@
-# kanban-capstone-js
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Kanban Javascript Project
+
+> Basic website that allows users to fetch data from an API end point. Displays it to the user and the user is able to comment and reserve the post. 
+
```

Comment:
- [ ] Please add a video that you and your partner should explain the project.

![mmmm](https://user-images.githubusercontent.com/61562626/143677714-bf745af2-89a4-422d-8744-fa560aff27be.jpeg)